apps/playground/src/style.css

```
@@ -0,0 +1,365 @@
+  --brand-soft: oklch(0.74 0.12 240 / 0.14);
+  --chart-fail: oklch(0.64 0.15 25);
+  --chart-warn: oklch(0.76 0.1 80);
+  --chart-pass: oklch(0.7 0.11 158);
+  --font-sans: "Geist", "Geist Fallback", ui-sans-serif, system-ui, sans-serif;
+  --font-mono: "Geist Mono", ui-monospace, monospace;
+  --text-label: 11px;
+  --text-meta: 12px;
+  --text-body: 13px;
+  --text-title: 14px;
+  --tracking-label: 0.08em;
+  color-scheme: dark;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)